x-pack/plugins/ml/public/application/components/ml_inference/components/on_failure_configuration.tsx

```diff
@@ -0,0 +1,220 @@
+                  defaultMessage="The {onFailure} configuration shown will be used as a default. It is used to specify a list of processors to run immediately after a processor failure and provides information on why the failure occurred. {onFailureDocsLink}"
+                  values={{
+                    onFailure: <EuiCode>{'on_failure'}</EuiCode>,
+                    onFailureDocsLink: (
+                      <EuiLink external target="_blank" href={links.ingest.pipelineFailure}>
+                        Learn more.
+                      </EuiLink>
+                    ),
+                  }}
+                />
+              </p>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={7}>
+            <EuiFlexGroup direction="column">
+              <EuiFlexItem>
+                <EuiFormRow
+                  fullWidth
+                  labelAppend={
+                    <EuiFlexGroup gutterSize="xs" justifyContent="flexStart">
+                      <EuiFlexItem grow={false}>
+                        <EuiButtonEmpty
+                          iconType="pencil"
+                          size="xs"
+                          onClick={() => {
+                            setEditOnFailure(!editOnFailure);
+                          }}
+                        >
+                          {editOnFailure ? cancelEditMessage : editMessage}
+                        </EuiButtonEmpty>
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        {editOnFailure ? (
+                          <SaveChangesButton
+                            onClick={updateOnFailure}
+                            disabled={isOnFailureValid === false}
+                          />
+                        ) : null}
+                      </EuiFlexItem>
+                      <EuiFlexItem grow={false}>
+                        {editOnFailure ? (
+                          <EuiButtonEmpty size="xs" onClick={resetOnFailure}>
+                            {i18n.translate(
+                              'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.resetOnFailureButton',
+                              { defaultMessage: 'Reset' }
+                            )}
+                          </EuiButtonEmpty>
+                        ) : null}
+                      </EuiFlexItem>
+                    </EuiFlexGroup>
+                  }
+                  helpText={
+                    <FormattedMessage
+                      id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.advanced.onFailureHelpText"
+                      defaultMessage="In the case of failure, the above configuration will set the index document as failed, provide the timestamp at which ingest failed, and provide context for the failure."
```

Comment:
```suggestion
                      defaultMessage="In case of failure, this configuration sets the index document as failed, provides the timestamp at which ingest failed, and the context for the failure."
```

Reply:
Updated in b0aa5e0d8ecc48b474dcd709982536fe852dcd9a